src/main/java/com/epam/java/pdp/figures/Shape.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.java.pdp.figures;
+import com.epam.java.pdp.figures.Point;
+
+public class Shape {
+    static int x1 = 1;
+    static int x2 = 2;
+    static int x3 = 3;
+    static int y1 = 2;
+    static int y2 = 1;
+    static int y3 = 3;
+    static Point p1 = new Point(x1, y1);
+    static Point p2 = new Point(x2, y2);
+    static Point p3 = new Point(x3, y3);
+    static Line ln1 = new Line(p1,p2);
+    static Line ln2 = new Line(p2,p3);
+    static Line ln3 = new Line(p3,p1);
+    static double a = ln1.lineLength();
+    static double b = ln2.lineLength();
+    static double c = ln3.lineLength();
+
+   public static void main (String[] args){
+       System.out.println("Line lengths are: ");
+       System.out.println(a);
+       System.out.println(b);
+       System.out.println(c);
+
+       isTriangle();
```

Comment:
Please use OOP. Introduce some new class called Shape for instance and delegate to it perform those methods

Reply:
there is a Shape class already. Should I create a new new class called like ShapeMethods and move all methods to this class?